### PR TITLE
[DOCS] update VisibilityNotifier(2D) class ref

### PR DESCRIPTION
**Pull request by cbscribe**

### CODE REVIEW

File: doc/classes/VisibilityNotifier.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisibilityNotifier" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Detect when the node is visible on screen.
+		Detects when the node is visible on screen.
 	</brief_description>
 	<description>
-		The VisibilityNotifier is used to notify when its bounding box enters the screen, is visible on the screen, or when it exits the screen.
+		he VisibilityNotifier detects when it is visible on the screen.  It also notifies when its bounding rectangle enters or exits the screen or a [Camera]'s view.
```
Comment (djrm) [inline doc/classes/VisibilityNotifier.xml]: Missing T

File: doc/classes/VisibilityNotifier.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisibilityNotifier" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Detect when the node is visible on screen.
+		Detects when the node is visible on screen.
 	</brief_description>
 	<description>
-		The VisibilityNotifier is used to notify when its bounding box enters the screen, is visible on the screen, or when it exits the screen.
+		The VisibilityNotifier detects when it is visible on the screen.  It also notifies when its bounding rectangle enters or exits the screen or a [Camera]'s view.
```
Comment (Noshyaar) [inline doc/classes/VisibilityNotifier.xml]: `screen.  It also` Additional space here

File: doc/classes/VisibilityNotifier2D.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisibilityNotifier2D" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Detect when the node is visible on screen.
+		Detects when the node is visible on screen.
 	</brief_description>
 	<description>
-		The VisibilityNotifier2D is used to notify when its bounding rectangle enters the screen, is visible on the screen, or when it exits the screen.
+		The VisibilityNotifier2D detects when it is visible on the screen.  It also notifies when its bounding rectangle enters or exits the screen or a viewport.
```
Comment (Noshyaar) [inline doc/classes/VisibilityNotifier2D.xml]: Also here

File: doc/classes/VisibilityNotifier2D.xml [hunk 4]
```diff
@@ -37,6 +37,7 @@
 	</methods>
 	<members>
 		<member name="rect" type="Rect2" setter="set_rect" getter="get_rect">
+			The VisibilityNotifier2D's bounding rectangle.
```
Comment (Noshyaar) [inline doc/classes/VisibilityNotifier2D.xml]: Additional space at the end

Comment (cbscribe) [reply doc/classes/VisibilityNotifier2D.xml]: Removed.

**Pull request merged**